vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -363,13 +363,14 @@ func (feeder *clusterStateFeeder) LoadPods() {
 			feeder.clusterState.DeletePod(key)
 		}
 	}
+	feeder.clusterState.VpasWithMatchingPods = make(map[model.VpaID]bool, len(feeder.clusterState.Vpas))
```

Comment:
I don't like this, it looks like we're exposing internals of feeder.clusterState here. OTOH we do it already with other parts of the state so maybe it's not really actionable.